src/components/Tree/TreeItem.spec.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import sinon from 'sinon';
+import assert from 'assert';
+import { shallow } from 'enzyme';
```

Comment:
Hey Libo, for tests we are trying to use `@testing-library/react` now. https://testing-library.com/docs/react-testing-library/intro/ You can find a few examples of how to use it in the react-gears code base

Reply:
Will update. What's the rational on migrating to testing-library/react from enzyme?